pkg/monitor/.dockerignore

```diff
@@ -0,0 +1,4 @@
+eve-monitor-rs/target/
```

Comment:
don't we need to have these on .gitignore as well?

Reply:
@rene we could. this is for development only. usually you do not have `eve-monitor-rs` under `pkg/monitor` unless you bind mount it for development. I can add it to `.gitignore`

Reply:
@rene added